aiojenkins/jenkins.py

```diff
@@ -278,11 +286,7 @@ async def get_version(self) -> JenkinsVersion:
 
         versions = header.split('.')
 
-        # no patch version
-        if len(versions) == 2:
-            versions.append('0')
-
-        return JenkinsVersion(*map(int, versions))
+        return make_jenkins_version(*versions)
```

Comment:
I suggest  here to mock aiohttp instead of function mock.

Or I can do it myself it you don't mind.